action.yml

```
@@ -27,23 +30,29 @@ runs:
     env:
       REPO: ${{ inputs.repo }}
     run: |
-      docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
       if [ -z "${{ inputs.tag_list }}" ]; then
-        tag_list=$docker_repo:$REPO,$docker_repo:$(date +%Y%m%d-%H%M)
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tag_list=()
+        for image_tag in "$REPO" "$REPO-$(date +%Y%m%d-%H%M)"; do
+          tag_list+=("${{ inputs.registry_url }}/$docker_repo":"$image_tag")
+        done
+        IFS=,
+        echo "::set-output name=taglist::${tag_list[*]}"
       else
-        tag_list=${{ inputs.tag_list }}
+        tag_list=$(echo ${{ inputs.tag_list }} | sed -E "s/([^,]+)/${{ inputs.registry_url }}\/\1/g")
+        echo "::set-output name=taglist::$tag_list"
       fi
-      echo "::set-output name=taglist::$tag_list"
     shell: bash
 
- - name: Set up Docker Buildx
+  - name: Set up Docker Buildx
```

```suggestion
 - name: Set up Docker Buildx
```

I think an extra space snuck in here